docs/source/releases/v1_12_2a0.rst

```diff
@@ -10,18 +10,19 @@
  | # # # for more details. If you did not receive the license, for more information see:
  | # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
 
-Version 1.12.2a0 (2024-02-21)
+Version 1.12.2a0 (2024-02-19)
 *****************************
 
+* Enhancements
 
+  * Initial Stage of the GeoIPS CLI
+  * CLI Stage Two - new implementation of 'geoips run' and more args for 'geiops list'
```

Comment:
Not ready for merge until #444 has been merged.

Reply:
But I will say that this PR is ready for review. All functionality has been added and all tests pass.

Reply:
This has been merged with the most recent updates to #444 and is ready for reviewal / merge process.